conf/config-default.yaml

```diff
@@ -14,122 +14,121 @@
     enable: true
-    listen:                       # APISIX listening port in https.
+    listen:                                      # APISIX listening port for HTTPS traffic.
       - port: 9443
         enable_http2: true
-    #   - ip: 127.0.0.3           # Specific IP, If not set, the default value is `0.0.0.0`.
-    #     port: 9445
-    #     enable_http2: true
-    #ssl_trusted_certificate: /path/to/ca-cert  # Specifies a file path with trusted CA certificates in the PEM format
-                                                # used to verify the certificate when APISIX needs to do SSL/TLS handshaking
-                                                # with external services (e.g. etcd)
-    ssl_protocols: TLSv1.2 TLSv1.3
+      # - ip: 127.0.0.3                          # If not set, default to `0.0.0.0`.
+      #   port: 9445
+      #   enable_http2: true
+    # ssl_trusted_certificate: /path/to/ca-cert  # Path to CA certificates in the PEM format.
+    ssl_protocols: TLSv1.2 TLSv1.3               # TLS versions supported.
     ssl_ciphers: ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384
-    ssl_session_tickets: false              #  disable ssl_session_tickets by default for 'ssl_session_tickets' would make Perfect Forward Secrecy useless.
-                                            #  ref: https://github.com/mozilla/server-side-tls/issues/135
+    ssl_session_tickets: false  # If true, session tickets are used for SSL/TLS connections.
+                                # Disabled by default because it renders Perfect Forward Secrecy (FPS)
+                                # useless. See https://github.com/mozilla/server-side-tls/issues/135.
+
+    key_encrypt_salt:           # Salt for SSL/TLS private key used with AES-128-CBC.
+      - edd1c9f0985e76a2        # Each salt value should be a hexadecimal string of length 16.
+                                # Support multiple salt values for rotation.
+                                # CAUTION: DO NOT MODIFY SALT VALUE AFTER KEY IS WRITTEN TO ETCD.
+                                # VALUES CANNOT BE DECRYPTED OTHERWISE.
 
-    key_encrypt_salt:             #  If not set, will save origin ssl key into etcd.
-      - edd1c9f0985e76a2          #  If set this, the key_encrypt_salt should be an array whose elements are string, and the size is also 16, and it will encrypt ssl key with AES-128-CBC
-                                  #  !!! So do not change it after saving your ssl, it can't decrypt the ssl keys have be saved if you change !!
-                                  #  Only use the first key to encrypt, and decrypt in the order of the array.
+    # fallback_sni: "my.default.domain"      # Fallback SNI to use if client does not send SNI during
+    #                                        # the handshake.
```

Comment:
Fallback SNI to be used in case the client does not send SNI during the handshake.